### ISSUE TITLE
Add open source projects to our portfolio

### ISSUE THREAD

**Issue by johndgiese** [innolitics/innolitics.github.io#10]

**Comment by johndgiese** [comment]
@andrewemark do you think we should add open source stuff inline with our company work, or in a new section below company work?

**Comment by johndgiese** [comment]
@ReeceStevens we should start with the natural neighbor project.  Once that is done, we should add the dicom-numpy repository.  Having nice looking images is key.

**Comment by andrewemark** [comment]
I think it should be a section below our company work. This gives us an opportunity to put a "show more" link below the contracted work (where we can put all the projects that aren't medical), followed by the "Open Source Projects" section. What do you think @johndgiese @ReeceStevens

**Comment by johndgiese** [comment]
Sounds good.  So perhaps leave the page title of "Portfolio" then below it have a subtitle "Client Projects", and then "Open Source Projects" below?

**Comment by andrewemark** [comment]
Hmm, perhaps. We could also just put "Client Projects" and "Open-source Projects" and eliminate the Portfolio heading. It seems a bit redundant?

**Comment by andrewemark** [comment]
We can also have the nav bar "stick" to the page you're on.

**Comment by johndgiese** [comment]
I like that; if we make the nav bar stick to the page you are on, this becomes a non-issue.

**Comment by andrewemark** [comment]
I think we can load the whole portfolio, and the show more link will be some simple JS to reveal the extra rows. Maybe cap it at the first 6 or 9 items before putting the "Show More"?

**Comment by johndgiese** [comment]
Do we even need the "Show More" button right now?  I can see once we have a lot of projects, but it seems like we could just load all of them for now.

**Comment by andrewemark** [comment]
I was thinking we could hide those that are not Medical by default.

**Comment by ReeceStevens** [comment]
It might be nice to have a "show more" button so that we can limit the displayed "client projects" to just medical sector related ones.

**Comment by ReeceStevens** [comment]
That also puts open-source projects that have medical-sector implications next to our medical-sector client work

**Comment by johndgiese** [comment]
I see, that makes sense to me.

**Comment by johndgiese** [comment]
Good idea!

**Comment by johndgiese** [comment]
Nice work @ReeceStevens !
